### DISCUSSION TITLE
Imporving product description field by rich text editor.

### DISCUSSION THREAD

**Discussion by shamun-khatri** [medusajs/medusa#8022] [Feature Requests]
Improving product description field by rich text editor where we can format text and add images like Shopify and Wordpress have.

![image](https://github.com/medusajs/medusa/assets/100433320/c8e8004f-8cbd-4251-841a-d8973663a216)

**Comment by Saipatnana** [comment]
yes i want to do this in my project can you help to do that

**Comment by shamun-khatri** [comment]
can we take this feature as a priority?
